examples/serve-side-encryption/README.md

```diff
@@ -0,0 +1,39 @@
+# Example: `server-side-encryption`
+
```

Comment:
missing letter in directory name: "serve-side-encryption"

Reply:
Eagle-eyed! Thank you